aws-lambda/README.md

```diff
@@ -39,25 +39,49 @@ SignalFx enables you to monitor the health and performance of your Lambda functi
 
 ### INSTALLATION
 
+#### SignalFx Wrapper vs Cloudwatch monitoring
+
+While you can use both the SignalFx Wrapper and Cloudwatch methods to monitor Lambda functions at the same time, you also have the option to use only one method. 
+
+Before you select a single method (or both), consider the following statements: 
+
+- Each method sends different metrics and uses different properties to uniquely identify a Lambda function.
+  - The SignalFx Wrapper sends real-time monitoring data of Lambda functions, as well as custom metrics.
+  - All Cloudwatch metrics available in your account are imported. 
```

Comment:
I do not know if the second statement "-All Cloudwatch metrics available in your account are imported" is relevant here. Line 49 also seems to be a bit out of context here? They also seem to be indented, I don't know if this was on purpose.

Reply:
No problem; I can remove it. 

I simply wanted to add statements to help support, "Each method sends different metrics and uses different properties to uniquely identify a Lambda function." Because as a reader, if I saw that statement, my next question would be, "What are the different metrics and properties used for each method?" But I guess that can be discovered reading further down the document. 

And yes, the indentation was intentional.